openedx/features/edly/utils.py

```diff
@@ -261,3 +265,27 @@ def send_comments_reply_email_to_comment_owner(comment, context):
     if parent_comment.user_id != comment.thread.user_id and parent_comment.user_id != comment.user_id:
         recipients = [parent_comment.user_id]
         send_bulk_mail_to_students.delay(recipients, message_context, 'comment_reply')
+
+
+def send_contact_us_email(data):
+    recipient_list = settings.CONTACT_US_SUPPORT_EMAILS
+    context = {
+        'requester_name': data['requester']['name'],
+        'requester_email': data['requester']['email'],
+        'body': data['comment']['body'],
+        'custom_fields': data['custom_fields']
+    }
+    payload = json.dumps(data)
+    try:
+        send_mail(
```

Comment:
I think we should use[ edx_ace for sending](https://github.com/edx/edx-platform/blob/f6603cb8cfd1f46236db6049516f7f8c6e1c38ab/lms/djangoapps/verify_student/management/commands/send_verification_expiry_email.py#L165) email instead of using django's `send_mail`

Reply:
done.